packages/darklang/stdlib/process.dark

```diff
@@ -5,17 +5,12 @@ module Darklang =
       /// The result of running a process, typically with `Process.run`
       type Result =
         { exitCode: Int
-          stdout: Bytes
-          stderr: Bytes }
 
-      // Return Ok stdout+stderr, or Error exitCode+stdout+stderr
-      let runCommand
-        (command: String)
-        (input: String)
-        : PACKAGE.Darklang.Stdlib.Result.Result<String, Result> =
-        let result = Builtin.Process.run command input
+          // TODO bytes, not strings -- see internal note in Process.fs
+          stdout: String
+          stderr: String }
 
-        if result.exitCode == 0 then
-          PACKAGE.Darklang.Stdlib.Result.Result.Ok(stdout ++ stderr)
-        else
-          PACKAGE.Darklang.Stdlib.Result.Result.Error result
```

Comment:
This was trying to be a bit fancy, but didn't work (and _won't work_ even more once stdout and stderr are Bytes again) and I suspect there's a better way to handle this. Made it less fancy, and the package fn is a simple pass-through now